tools/pd-ctl/tests/config/config_test.go

```diff
@@ -1144,6 +1144,34 @@ func (suite *configTestSuite) checkMicroServiceConfig(cluster *pdTests.TestClust
 	re.False(svr.GetMicroServiceConfig().EnableSchedulingFallback)
 }
 
+func (suite *configTestSuite) TestRegionRules() {
```

Comment:
Why add it in this PR?

Reply:
From other unmerged pr

Reply:
Then, we should remove it?

Reply:
Or opening a new pr for adding test to cover it? They are both acceptable for me.